configs/_base_/default_runtime.py

```diff
@@ -5,7 +5,9 @@
     timer=dict(type='IterTimerHook'),
     logger=dict(type='LoggerHook', interval=50),
     param_scheduler=dict(type='ParamSchedulerHook'),
-    checkpoint=dict(type='CheckpointHook', interval=1),
+    checkpoint=dict(
+        type='CheckpointHook', interval=1, save_best='coco/AP',
+        rule='greater'),
```

Comment:
`default_runtime.py` will be used by all experiments. It's better not to add coco settings here.

Reply:
Agreed, I will move it to coco configs.